src/tests/ConfigureRecordingSetTests.test.tsx

```diff
@@ -0,0 +1,176 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Are these linter suppression necessary?

Reply:
I think I can get rid of the first two, I do actually need the last two though - the empty function suppression is so i can pass empty functions into the component I'm testing (no need to mock them since they aren't being used), and the prefer-expect-assertions has to be disabled for react-testing-library to work with the async calls

Reply:
Empty function -> There is a `noOp()` function I defined in one of the utils.

Other things sound good.